README.md

```diff
@@ -164,6 +164,98 @@ make install
 
 </details>
 
```

Comment:
More a general term, the support for ESSL and ArmPL is not documented in the README file. We should fix that I think.

Reply:
There is no "Supported Libraries" section in general. The requirements section names clfft, cufft and fftw, rocfft is only mentioned in the issues, mkl isn't found anywhere.